_delphi_utils_python/README.md

```diff
@@ -22,13 +22,20 @@ Source code can be found here:
 
 ## Logger Usage
 
+### Commonly used argument names:
```

Comment:
Please also add this text to [`logger.py`](https://github.com/cmu-delphi/covidcast-indicators/blob/d52dcd5a6b84083fb5cbca4164ebdd8429ac81d3/_delphi_utils_python/delphi_utils/logger.py#L5-L8), replacing the outdated instructions there already.

```suggestion
To make our structured logging as useful as it can be, particularly within the context of how we use logs in Elastic, the `event` argument (typically the first unnamed arg) should be a static string (to make filtering easier), and each dynamic/varying value should be specified in an individual meaningfully- and consistently-named argument to the logger call (for use in filtering, thresholding, grouping, visualization, etc).

### Commonly used argument names:
```